### PR TITLE
Meta data rules form

### PR DESCRIPTION
**Pull request by bhatiadheeraj**

### CODE REVIEW

File: ui/src/store/index.ts [hunk 1]
```diff
@@ -444,6 +447,42 @@ function loadDatatype(
...
+/** 
+ * 
+ * function LoadMetaDataRules() {
+ * 2 files loaded -> 1st perf/asl -> asl.yaml and convert it to above json
+ * pet -> pet.yaml 
+ * 
+ * }
+*/
+
+async function LoadMetaDataRules() {
```
Comment (bhatiadheeraj) [inline ui/src/store/index.ts:474]: @dlevitas we can get rid of it right ? I guess we tested it while exploring

File: ui/src/SeriesPage.vue [hunk 2]
```diff
@@ -182,13 +188,16 @@ import { defineComponent } from 'vue'
...
+import ModalityForm from './components/modalityForm.vue'
 
 import { prettyBytes } from './filters'
 
 import { Series, IObject } from './store'
 
 import { validate_Entities_B0FieldIdentifier_B0FieldSource } from './libUnsafe'
-
+import aslYaml from "../src/assets/schema/rules/sidecars/asl.yaml";
+import petYaml from '../src/assets/schema/rules/sidecars/pet.yaml';
```
Comment (bhatiadheeraj) [inline ui/src/SeriesPage.vue:199]: Note for myself : 

Remove pet as we are not more using it

File: ui/src/SeriesPage.vue [hunk 3]
```diff
@@ -206,12 +215,19 @@ export default defineComponent({
...
+            rules: {},
+            formData: {
+            },
         }
     },
 
     computed: {
         ...mapState(['ezbids', 'bidsSchema', 'config']),
-        ...mapGetters(['getBIDSEntities', 'getBIDSMetadata', 'getURL']), //doesn't work with ts?
+        ...mapGetters(['getBIDSEntities', 'getBIDSMetadata', 'getURL', 'getMetaDataRule']), //doesn't work with ts?
```
Comment (bhatiadheeraj) [inline ui/src/SeriesPage.vue:230]: @dlevitas we should get rid of it right 'getMetaDataRule'?

File: ui/src/store/index.ts [hunk 4]
```diff
@@ -13,7 +13,7 @@ import {
     SchemaTypeLike,
 } from '../../../bids-validator/bids-validator/src/types/schema'
 import { Severity } from '../../../bids-validator/bids-validator/src/types/issues'
-
+import * as yaml from 'js-yaml';
```
Comment (bhatiadheeraj) [inline ui/src/store/index.ts:16]: should we get rid of it ?

File: ui/src/store/index.ts [hunk 5]
```diff
@@ -340,6 +340,7 @@ const state = {
 
         defacingMethod: "",
         includeExcluded: true,
+        sideCar: {} as {[key: string]: any},
```
Comment (bhatiadheeraj) [inline ui/src/store/index.ts:343]: should we get rid of it ?
@dlevitas

File: ui/src/store/index.ts (hunk 4 above)
Comment (dlevitas) [reply ui/src/store/index.ts:16]: yes, that line is no longer needed

File: ui/src/store/index.ts (hunk 5 above)
Comment (dlevitas) [reply ui/src/store/index.ts:343]: Yes, this field is unnecessary now

**Pull request merged**